src/modules/ItemList.js

```diff
@@ -0,0 +1,143 @@
+import { createListItem, ItemElem } from './ListItemMod.js';
+import icon from '../asset/resource/icons.svg';
+
+const xlinkHref = [`${icon}#icon-check`, './asset/resource/icons.svg#icon-more-vert',
```

Comment:
I suggest you use more clear variable names that tell their purpose to avoid confusion and to make your code easily maintainable

Reply:
Although I would love to change the href name structure, I cannot. Href naming has the following structure:
- First, the relative or absolute reference
- Follow by the hashtag symbol '#'
- Finally, the ID of the specific group of paths inside the SVG files.